backend/service/services.go

```diff
@@ -64,8 +69,18 @@ func InitStatsD(statsdURL string, log *logrus.Logger) *statsd.Client {
 	return client
 }
 
-func InitDB(databaseURL string, log *logrus.Logger) *pgx.Conn {
-	connection, err := pgx.Connect(context.Background(), databaseURL)
+func InitPgDB(pgdbURL string, log *logrus.Logger) *pgx.Conn {
+	connection, err := pgx.Connect(context.Background(), pgdbURL)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return connection
+}
+
+func InitMsDB(msdbURL string, log *logrus.Logger) *mssql.DB {
+	// connection string with placeholders user, password, database
+	connString := "server=" + msdbURL + ";user id=user;password=password;database=database"
```

Comment:
This crates a [connString (DSN)](https://www.connectionstrings.com/sql-server/) which should instead be fully passed as a [flag in main](https://github.com/elonsoc/ods/blob/a0b11b1451069d04f66bf3401662214f9f83e584/backend/main.go).
```suggestion

```

Reply:
should we define this connection string in main or is there no need since both mssql and postgres connections are passed as flags in main.go?

Reply:
we shouldn't define this as this will be passed into the program through the environment variables or flags.